Apps/WebClient/src/ClientApp/src/components/DependentCardComponent.vue

```diff
@@ -1341,6 +1447,119 @@ export default class DependentCardComponent extends Vue {
                         </b-card>
                     </div>
                 </b-tab>
+                <b-tab
+                    v-if="isClinicalDocumentTabShown"
+                    :disabled="isExpired"
+                    no-body
+                    :data-testid="`clinical-document-tab-${dependent.ownerId}`"
+                    @click="fetchClinicalDocuments"
+                >
+                    <template #title>
+                        <div
+                            :data-testid="`clinical-docuemnt-tab-title-${dependent.ownerId}`"
```

Comment:
Typo: docuemnt should be document.  (Applies to references in functional tests as well.)

Reply:
Fixed